packages/file_selector/file_selector_windows/windows/file_selector_plugin.cpp

```diff
@@ -111,7 +86,7 @@ class DialogWrapper {
 
   // Attempts to set the default folder for the dialog to |path|,
   // if it exists.
-  void SetDefaultFolder(const std::string& path) {
+  void SetDefaultFolder(const std::string_view& path) {
```

Comment:
Prefer passing string_view by value rather than ref since they're teensy and fit in a couple registers.

Reply:
Oops, I know better but got careless about updating things. Thanks for catching that!